templates/layout.html

```diff
@@ -12,17 +12,52 @@
     <link rel="stylesheet" href="{{ url_for('static', filename='css/custom_reporting.css') }}"crossorigin="anonymous"/>
 
     <title>{% block title %}{% endblock %}</title>
+    {% if config["FLASK_ADMIN_SWATCH"] == "slate" -%}
```

Comment:
dont do if/else statements like this on the html elements just for themes

something like will be much easier to manage:
```html
<link rel="stylesheet" href="{{ url_for('static', filename='css/custom_reporting.css') }}"crossorigin="anonymous"/>
{% if config['CSS_THEME'] %}
    <link rel="stylesheet" href="{{ url_for('static', filename=config['CSS_THEME']) }}"crossorigin="anonymous"/>
{% endif %}
```

Reply:
Hmmm.... the custom_reporting.css file has got nothing to do with this particular page. The nav class used here is what is in the examples from the bootswatch, not the custom_reporting.css.

Reply:
ive tried adding the navbar to the custom_reporting.css files but i cant get it to work at all. so i cant figure a way of getting removing if condition for the.

Reply:
im applying the config['CSS_THEME'] but cant figure a way to do this with the navbar part.

Reply:
you dont need to change the bootstrap templates you can control them with the FLASK_ADMIN_SWATCH param (https://github.com/flask-admin/flask-admin/blob/e39f786374ce0e60db93f583efacb4672de0025c/flask_admin/templates/bootstrap3/admin/base.html#L15)

As for the changing the navbar bootstrap styles if you really want to change them you could do something like:

```html
...
<nav class="navbar navbar-expand-lg navbar-{{ config['CSS_THEME_INTENSITY'] }} bg-{{ config['CSS_THEME_INTENSITY'] }}">
...
```

Reply:
changed this so its using the intensity.